.github/workflows/release.yml

```diff
@@ -93,13 +98,28 @@ jobs:
           profile: minimal
           target: ${{ matrix.target }}
 
+      - name: Setup | Install cargo-wix [Windows]
+        # aarch64 is only supported in wix 4.0 development builds
+        if: matrix.os == 'windows-latest' && matrix.target != 'aarch64-pc-windows-msvc'
+        run: cargo install --version 0.3.2 cargo-wix
```

Comment:
It's unfortunate that this, being a hard-coded version, won't be updated by Dependabot.
I wish we could have cargo subcommands be retrieved from our dev-dependencies. That's not possible though, right?

Reply:
I don't think that's possible.